chat-demo-android/src/main/AndroidManifest.xml

```diff
@@ -81,5 +82,14 @@
 
         <meta-data android:name="com.google.android.gms.version"
     		android:value="@integer/google_play_services_version" />
+
+        <meta-data
+            android:name="firebase_crashlytics_collection_enabled"
+            android:value="false" />
```

Comment:
Is this staying false even if you set fabricApiKey?

We probably must mention this in readme, if it needs to be changed.

Reply:
Yes, it should be staying false even if fabricApiKey is set. It just means "do not auto-start reporting". We have to start it manually from code in order to enable ndk crashes reporting

Reply:
Ok, good. Probably add a comment about it in the Manifest.

Reply:
comment added